Makefile

```diff
@@ -692,6 +692,30 @@ deploy/db:
 	@time timeout --foreground 3m bash -c "until $(OC) get pods -n $(NAMESPACE) | grep kas-fleet-manager-db | grep -v deploy | grep -q Running; do echo 'database is not ready yet'; sleep 10; done"
 .PHONY: deploy/db
 
+# Touch all the necessary files for fleet manager to start
```

Comment:
Which has been the criteria to decide which secrets to include here? The ones that are needed to not cause a crash of kfm at startup with the defaults in the `development` environment?

Reply:
I took the list from the go template and renamed the dinosaur ones to kafka.

Reply:
Then we should check for potentially missing secrets aside from the ones added in this PR, as the go template is not 100% synced with kfm repository